src/clusterfuzz/_internal/system/environment.py

```diff
@@ -365,10 +367,12 @@ def get_llvm_symbolizer_path():
 
   # Make sure that we have a default llvm-symbolizer for this platform.
   if not os.path.exists(llvm_symbolizer_path):
+    logs.info(f'Debug: None Symbolizer Path :- {llvm_symbolizer_path}')
```

Comment:
Maybe we should make a debug log function. I don't like writing "debug" in an info log function.

Reply:
Hi Jonathan, 
These logs are for investigating the root cause of ```AttributeError: 'NoneType' object has no attribute 'stdin'``` issue. Once we debug this issue, we will remove the logs.